lib/jazzy/podspec_documenter.rb

```diff
@@ -97,6 +97,23 @@ def self.github_file_prefix(podspec)
 
     private_class_method :github_file_prefix
 
+    # Latest valid value for SWIFT_VERSION.
+    LATEST_SWIFT_VERSION = '4.2'.freeze
```

Comment:
Should this be `4.2.1`?

Reply:
Nope, here we are working out values to pass to `swiftc -swift-version` via `xcodebuild`'s `SWIFT_VERSION`.  `4.2.1.` isn't valid for `swiftc` and if you pass it to `xcodebuild` it gets turned into `4`.